packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -160,7 +176,8 @@ class CpuProfileData {
 
       while (currentStackFrame != null) {
         stackFramesWithTag[currentId!] = currentStackFrame.shallowCopy(
-          copySampleCountsAndTags: false,
+          copySampleCounts: false,
+          profileMetaData: metaData,
```

Comment:
I thought we don't need each stack frame to store a copy of the metadata anymore?

Reply:
We do, unless we want to do some refactoring to remove its usage in `ProfilableDataMixin`.